release-engineering/role-handbooks/branch-manager.md

```diff
@@ -224,16 +248,44 @@ Prior to cutting a release version, [open a "Cut a Release" issue](https://githu
 
 On the issue template, there are comments describing the predefined items that need to be completed.
 
+For the item `Screenshot unhealthy release branch testgrid boards...`:
+
+You can use the `testgridshot` tool to create screenshots of unhealthy testgrid boards. `testgridshot` also takes care of uploading the screenshots to the `k8s-staging-releng` bucket, and generating the markdown table that you can post as a comment to the issue.
+
+`testgridshot` is a Bash script located in the [kubernetes/release] repository. By default, it takes a screenshots of **blocking** and **informing** testgrid boards that have jobs that are **failing**. If you want to customize this, open the script in your preferred text editor and see the comment at the beginning of the script for possible customizations.
+
+The script can be invoked in the following way:
+
+```shell
+./testgridshot <release>
+```
+
+`<release>` can be in form of `1.xy` (e.g. 1.19) or `master` (used when cutting alpha releases).
+
+Eventually, you may want to include **flaking** jobs in addition to the **failing** jobs. In that case, you can invoke `testgridshot` such as:
+
+```shell
+STATES='FAILING FLAKY' ./testgridshot <release>
+```
+
+Once the script generates the markdown table, post it as a comment on the created issue. You can take a look at the [following comment](https://github.com/kubernetes/sig-release/issues/1249#issuecomment-696702503) as an example.
+
 For the item `Collect metrics, links...`:
 
-You can make a copy of this spreadsheet to collect metrics of build times and include links to the GCP build console:
+You can post a thread in the [#release-management] Slack channel and include links to the GCP build console. You can take a look at the [following thread][example-release-thread] as an example.
+
+After the release process has been completed, you can use the `krel gcbmgr history` command to generate a markdown table with commands used to run the jobs, links to the GCB logs, and the information about each job.
 
-[Branch Management Log](https://docs.google.com/spreadsheets/d/1gbr2byg7AmGvVv6onZtgowa729KZcyYTyH82qn1czK8/edit?usp=sharing)
+```shell
+krel gcbmgr history --branch release-1.xy --date-from <date-of-release>
```

Comment:
I'm not sure is this command correct at all. Additionally, can this command be used for alpha releases, and if yes, how it would look like?

Reply:
this command gets the jobs that ran in the day you set for the specific branch and build the table, it does not matter if is alpha or any other type

Reply:
cc @cpanato

Reply:
Thank you for the explanation!